src/main/java/org/openstreetmap/atlas/checks/validation/relations/OneMemberRelationCheck.java

```diff
@@ -0,0 +1,83 @@
+package org.openstreetmap.atlas.checks.validation.relations;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.openstreetmap.atlas.checks.base.BaseCheck;
+import org.openstreetmap.atlas.checks.flag.CheckFlag;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasObject;
+import org.openstreetmap.atlas.geography.atlas.items.Relation;
+import org.openstreetmap.atlas.geography.atlas.items.RelationMember;
+import org.openstreetmap.atlas.tags.RelationTypeTag;
+import org.openstreetmap.atlas.utilities.configuration.Configuration;
+
+/**
+ * This check flags {@link Relation}s which contain only one member.
+ *
+ * @author savannahostrowski
+ */
+public class OneMemberRelationCheck extends BaseCheck
+{
+    public static final String OMR_INSTRUCTIONS = "This relation, {0,number,#}, contains only "
+            + "one member.";
+
+    private static final List<String> FALLBACK_INSTRUCTIONS = Collections
+            .singletonList(OMR_INSTRUCTIONS);
+
+    @Override
+    protected List<String> getFallbackInstructions()
+    {
+        return FALLBACK_INSTRUCTIONS;
+    }
+
+    public OneMemberRelationCheck(final Configuration configuration)
+    {
+        super(configuration);
+    }
+
+    @Override
+    public boolean validCheckForObject(final AtlasObject object)
+    {
+        return object instanceof Relation && !this.isFlagged(object.getOsmIdentifier());
```

Comment:
Is there a need to flag relation OSM identifiers? As far as I know there is one-to-one mapping between OSM relations and Atlas relations.

Reply:
No, you're right. I think that OSM relation identifiers and Atlas relation identifiers are one-to-one.

Reply:
You probably do not need to flag them then. Because you won't hit the same relation twice.